tools/snippets/unify_test_files.py

```diff
@@ -261,7 +269,8 @@ def num_to_underscore(m: re.Match) -> str:
     def add_client_type_to_function(self) -> None:
         """Includes the data type."""
         client_in_definition = r"(?:\bdef\b.*)\bclient\b"
-        if client_match := re.search(client_in_definition, self.line):
+        client_match = re.search(client_in_definition, self.line)
```

Comment:
these guys probably fell afoul of the `test_python_support` and should be reverted to use the walrus